lib/SAMLSettings.php

```diff
@@ -167,6 +166,7 @@ public function getOneLoginSettingsArray(int $idp): array {
 					'url' => $this->configurations[$idp]['idp-singleSignOnService.url'] ?? '',
 				],
 				'x509cert' => $this->configurations[$idp]['idp-x509cert'] ?? '',
+				'passthroughParameters' => $this->configurations[$idp]['idp-passthroughParameters'] ?? '',
```

Comment:
is this now correct with this 'idp-' text prefix?

Reply:
I now have this code running in 8 differnt sites, all working, so I think we are good.

Reply:
Yes, all good, boils down to the fact that we have to convert it to a php-saml compatible array structure. Hence the toying with the prefix.